src/app/components/candidate-card-expanded/candidate-card-expanded.component.html

```diff
@@ -103,7 +103,7 @@ <h2>{{ candidate['description'] }}</h2>
   
         <ng-container matColumnDef="percentage">
           <th mat-header-cell *matHeaderCellDef>Percentage</th>
-          <td [ngClass]="{'remove-border': i == dataSource.data.length - 1}" mat-cell *matCellDef="let element; let i = index">{{ element.percentage | percent }}</td>
+          <td [ngClass]="{'remove-border': i == dataSource.data.length - 1}" mat-cell *matCellDef="let element; let i = index">{{ element.percentage/100 | percent }}</td>
```

Comment:
```suggestion
          <td [ngClass]="{'remove-border': i == dataSource.data.length - 1}" mat-cell *matCellDef="let element; let i = index">{{ element.percentage | number:'1.0-0' }}%</td>
```

This change should resolve the merge conflict.